locobot/robot/slam_pkg/README.md

```diff
@@ -13,14 +13,17 @@ roscore &
 
 ```
 load_pyrobot_env
-python slam.py --robot habitat --goal 10 2 0 --map_size 4000 --robot_rad 25 --vis --save_vis 
+python slam.py --robot habitat --goal 10 2 0 --map_size 4000 --robot_rad 25 --vis --save_vis --store_path ./tmp --dataset_path /Replica-Dataset --scene apartment_0
 ```
 
  Results should look something like this
 <p align="center">
     <img src="https://media.giphy.com/media/eTof1wrCZHXw83tGwH/giphy.gif", width="960" height="192">
 </p>
- 
+
+- `save_vis` arg will also store `rgb, depth, seg, robot trajectory(data.json)` under `store_path(args)/scene(args)` folder
```

Comment:
how are `--vis` and `--save_vis` different?

Reply:
`--vis` will show plt window while robot is exploring, this is helpful when there is screen attached to robot
`--save_vis` will store exploration visualization in addition to rgbd & seg data